.dotnet/src/Custom/Batch/BatchOperationToken.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.ClientModel;
+using System.Diagnostics;
+using System.IO;
+using System.Text.Json;
+
+#nullable enable
+
+namespace OpenAI.Batch;
+
+internal class BatchOperationToken : ContinuationToken
+{
+    public BatchOperationToken(string batchId)
+    {
+        BatchId = batchId;
+    }
+
+    public string BatchId { get; }
+
+    public override BinaryData ToBytes()
+    {
+        using MemoryStream stream = new();
+        using Utf8JsonWriter writer = new(stream);
+        writer.WriteStartObject();
+
+        writer.WriteString("batchId", BatchId);
+
+        writer.WriteEndObject();
+
+        writer.Flush();
+        stream.Position = 0;
+
+        return BinaryData.FromStream(stream);
```

Comment:
Do we need to use memory stream and JSON writer to produce this token? Can we simply return `$"\{{batchId}\}"`

Reply:
Saving this to come back to in a separate PR